deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -2114,6 +2122,20 @@ extension ChatViewController {
         navigationController?.present(inputDlg, animated: true, completion: nil)
     }
 
+    func didTapVcard(msg: DcMsg) {
+        guard let file = msg.file else { return }
+        guard let vcards = dcContext.parseVcard(path: file) else { return }
+        guard let vcard = vcards.first else { return }
```

Comment:
We could combine these three `guard` in one, like:

```suggestion
        guard let file = msg.file,
              let vcards = dcContext.parseVcard(path: file),
              let vcard = vcards.first else { return }
```